plugins/topology/src/components/Topology/TopologySideBar/TopologyResourcesTabPanel.tsx

```diff
@@ -83,10 +84,10 @@ const TopologyResourcesTabPanel = ({
                     additionalClassNames="hidden-xs"
                     noStatusBackground
                   >
-                    <Status status={pod.status?.phase ?? ''} />
+                    <Status status={podPhase(pod)} />
                   </ResourceStatus>
                 </span>
-                <span style={{ flex: '1' }}>
+                <span style={{ flex: '1', paddingLeft: '10px' }}>
```

Comment:
@debsmita1 I would suggest adding `gap` css style to `li` so that all items have consistent gap.
```
<li style={{gap: '10px'}} >...</li>
 ```